ch.protonmail.protonmail-bridge.yaml

```diff
@@ -7,23 +7,13 @@ separate-locales: false
 finish-args:
   - --share=ipc
   - --socket=x11
+  - --socket=wayland
```

Comment:
Why was this added?

Reply:
Tested it with `xeyes`.  Indeed, it does not use wayland, I will remove it in the next update. 

Just to clarify, is there an issue with the redundant socket enabled?

Reply:
No harm, just odd having unused permissions.

Reply:
Actually, there is harm, which I found very weird. 

I was unable to even move the window around with that flag enabled. I pushed a fix in #11.

Reply:
I remember why I added that socket, protonmail-bridge.{rpm,deb} bundles wayland libraries, which is weird considering it is not using wayland.